### PR TITLE
ui tests

### PR DESCRIPTION
**Pull request by AndreeaDora**

### CODE REVIEW

File: scripts/protractor/spec.js [hunk 1]
```diff
@@ -13,62 +43,141 @@ var sy = {
...
+    deleteProjectButton: by.id('delete-project-button'),
+    inspectViewBuilds: by.repeater('test in vm.results.testResults'),
+    inspectViewTestName: by.id('inspect-view-test-name'),
+    inspectViewMagnifyingGlass: by.id('inspect-view-test-results'),
+    inspectProjectGoToProjectsButton: by.id('inspect-go-to-projects')
 };
 
-// TODO: Look into how protractor *knows* when our app is pending tasks. Can we remove tha `browser.waits`?
+// TODO: Let's try to comment each step. Even with descriptive names, it's kind of hard to follow
+// TODO: missing case for removing projects/images/tests
```
Comment (soninob) [inline scripts/protractor/spec.js]: What is the plan for this? When are we adding missing tests?

File: scripts/protractor/spec.js [hunk 2]
```diff
@@ -77,54 +186,337 @@ describe('ILM', function() {
...
+     inspectButton.element(by.css('i[class="search icon"]')).click();
+     /!*browser.wait(protractor.ExpectedConditions.visibilityOf(element.all(sy.inspectViewBuilds).get(-1), 60000));
+     var lastBuild = element.all(sy.inspectViewBuilds).get(-1);
+     expect(lastBuild.element(sy.inspectViewMagnifyingGlass).getAttribute('class')).toBe('basic compact blue ui icon button');
+     element.all(sy.inspectViewBuilds).get(-1)
+     .element(sy.inspectViewMagnifyingGlass).click();
+     var testResultsHeader = element(by.css('.ui.header .content'));
+     browser.wait(protractor.ExpectedConditions.visibilityOf(testResultsHeader, 60000));
+     expect(testResultsHeader.getText()).toEqual('Clair report for image ' + config.imageName + ":" + config.tag);*!/
+     });*/
```
Comment (soninob) [inline scripts/protractor/spec.js]: If we are not using it, please remove it from the code.

File: scripts/protractor/spec.js (hunk 1 above)
Comment (AndreeaDora) [inline scripts/protractor/spec.js]: @soninob This tests were already added. I will remove the comment.
